************* Module distutils.sysconfig
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:43:0: C0103: Argument name "d" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:44:8: C0103: Variable name "fn" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:49:0: C0103: Constant name "_sys_home" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:52:4: C0103: Argument name "d" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:65:0: C0103: Constant name "python_build" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:71:0: C0103: Constant name "build_flags" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:74:8: C0103: Constant name "build_flags" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:100:8: R1706: Consider using ternary (BASE_EXEC_PREFIX if plat_specific else BASE_PREFIX) (consider-using-ternary)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:101:4: R1705: Unnecessary "elif" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:108:12: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:144:12: R1706: Consider using ternary (BASE_EXEC_PREFIX if plat_specific else BASE_PREFIX) (consider-using-ternary)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:146:12: R1706: Consider using ternary (EXEC_PREFIX if plat_specific else PREFIX) (consider-using-ternary)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:151:8: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:156:8: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:183:12: C0103: Constant name "_config_vars" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:183:12: W0603: Using the global statement (global-statement)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:186:16: C0415: Import outside toplevel (_osx_support) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:190:9: C0103: Variable name "cc" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:190:60: C0103: Variable name "ar" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:202:12: C0103: Variable name "cc" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:221:12: C0103: Variable name "ar" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:167:0: R0912: Too many branches (15/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:260:31: W0212: Access to a protected member _multiarch of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:264:0: C0103: Argument name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:264:0: C0103: Argument name "g" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:280:8: C0103: Variable name "m" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:282:12: C0103: Variable name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:282:15: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:283:17: C0321: More than one statement on a single line (multiple-statements)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:283:17: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:284:31: C0321: More than one statement on a single line (multiple-statements)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:287:12: C0103: Variable name "m" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:299:0: C0103: Argument name "fn" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:299:0: C0103: Argument name "g" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:299:0: R0914: Too many local variables (18/15) (too-many-locals)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:306:4: C0415: Import outside toplevel (distutils.text_file.TextFile) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:307:4: C0103: Variable name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:318:8: C0103: Variable name "m" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:320:12: C0103: Variable name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:320:15: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:321:12: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:329:20: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:346:12: C0103: Variable name "m" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:348:16: C0103: Variable name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:376:29: C0321: More than one statement on a single line (multiple-statements)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:343:4: R1702: Too many nested blocks (7/5) (too-many-nested-blocks)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:396:11: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:299:0: R0912: Too many branches (29/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:299:0: R0915: Too many statements (68/50) (too-many-statements)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:405:0: C0103: Argument name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:405:28: W0622: Redefining built-in 'vars' (redefined-builtin)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:421:8: C0103: Variable name "m" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:430:0: C0103: Constant name "_config_vars" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:443:4: C0103: Constant name "_config_vars" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:443:4: W0603: Using the global statement (global-statement)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:450:4: C0103: Variable name "g" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:463:4: C0103: Constant name "_config_vars" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:463:4: W0603: Using the global statement (global-statement)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:477:4: C0103: Constant name "_config_vars" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:477:4: W0603: Using the global statement (global-statement)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:492:8: C0103: Variable name "SO" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:530:12: C0415: Import outside toplevel (_osx_support) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:533:4: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:467:0: R0912: Too many branches (13/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:547:8: C0415: Import outside toplevel (warnings) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:18:0: W0611: Unused get_platform imported from util (unused-import)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:18:0: W0611: Unused get_host_platform imported from util (unused-import)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:13:0: C0411: standard import "import os" should be placed before "import _imp" (wrong-import-order)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:14:0: C0411: standard import "import re" should be placed before "import _imp" (wrong-import-order)
/log/firmware/firmware/usr/lib/python3.8/distutils/sysconfig.py:15:0: C0411: standard import "import sys" should be placed before "import _imp" (wrong-import-order)

-----------------------------------
Your code has been rated at 7.50/10

